docusaurus/docs/guides/deployments/10-linux/30-console.mdx

```diff
@@ -3,6 +3,40 @@ title: Console Deployment
 sidebar_label: Console
 ---
 
-import ConsoleReadme from '/_remotes/ziti-console/README.md';
+The console can be bound to the same web listener as the controller's management API. The console is then accessed by navigating to the controller's web address.
 
-<ConsoleReadme />
+1. On the controller host, download the latest release of the console from GitHub. You can use any console version >= 3.0.0.
+
+    ```text
+    wget https://github.com/openziti/ziti-console/releases/latest/download/ziti-console.zip
+    ```
+
+1. Unzip the console in the controller's working directory.
+
+    ```text
+    sudo unzip -d /var/lib/ziti-controller/zac ./ziti-console.zip
+    ```
+
+1. Add a new API binding `zac` to the same web listener where `edge-management` is bound. Set the `location`  option the relative path to unzipped console files.
+
+    ```text
+    
+    - binding: zac
+      options:
+        location: ./zac
```

Comment:
as we discussed live, were you going to use an abosolute path here and take any nuance about the 'relativity' of the path out of the equation?